django/db/backends/oracle/operations.py

```diff
@@ -676,6 +676,25 @@ def bulk_insert_sql(self, fields, placeholder_rows):
             for field in fields
             if field
         ]
+        if (
+            self.connection.features.supports_insert_multiple_rows
+            # A workaround with UNION of SELECTs is required for models without
+            # any fields.
+            and field_placeholders
+        ):
+            placeholder_rows_sql = []
+            for row in placeholder_rows:
+                placeholders_row = (
+                    field_placeholders[i] % placeholder
+                    for i, placeholder in enumerate(row)
```

Comment:
Any thoughts about using `zip` here? I assume they have the same length?

```suggestion
                    field_placeholder % placeholder
                    for field_placeholder, placeholder in zip(field_placeholders, row)
```

Reply:
Applied, thanks :+1: